patches/nginx-1.11.2-privilege_process.patch

```diff
@@ -0,0 +1,232 @@
+diff --git a/src/core/nginx.c b/src/core/nginx.c
+index 60f8fe7..b15f84c 100644
+--- a/src/core/nginx.c
++++ b/src/core/nginx.c
+@@ -53,6 +53,13 @@ static ngx_command_t  ngx_core_commands[] = {
+       offsetof(ngx_core_conf_t, master),
+       NULL },
+ 
++    { ngx_string("privilege_process"),
```

Comment:
We don't need an explicit config directive to enable this. We only need a C API for 3rd-party module to request at run time. Just like the existing cache manager and cache loader processes have no explicit config directives, but are triggered automatically by the presence of any http cache directives like `proxy_cache`.